src/test/java/org/zalando/planb/revocation/config/FallbackTokenInfoRequestExecutorTest.java

```diff
@@ -1,45 +1,85 @@
 package org.zalando.planb.revocation.config;
 
+import com.github.tomakehurst.wiremock.http.ContentTypeHeader;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.junit.Rule;
 import org.junit.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
 import org.springframework.web.client.HttpClientErrorException;
 import org.zalando.planb.revocation.AbstractOAuthTest;
 
 import java.util.Map;
+import java.util.UUID;
 
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class FallbackTokenInfoRequestExecutorTest extends AbstractOAuthTest {
 
-    private Integer wireMockPort = Integer.valueOf(System.getProperty("wiremock.port", "10080"));
+    @Rule
+    public WireMockRule secondServerMock = new WireMockRule(10081);
 
     @Test
     public void testSingleEndpointValidToken() {
         FallbackTokenInfoRequestExecutor executor =
-                new FallbackTokenInfoRequestExecutor("http://localhost:" + wireMockPort + "/tokeninfo");
+                new FallbackTokenInfoRequestExecutor("http://localhost:10080/tokeninfo");
```

Comment:
Yeah, was trying to keep it simple, but its true that's not the case if the environment is set to a different value.